<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirectNeighborDilationDialog</class>
 <widget class="QDialog" name="DirectNeighborDilationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1567</width>
    <height>997</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Direct Neighbor Dilation Dialog</string>
  </property>
  <widget class="QLabel" name="label_directDilationOrigImg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>531</width>
     <height>431</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_directDilationMaskImg">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>550</y>
     <width>421</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_directDilationRecImg">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>30</y>
     <width>411</width>
     <height>381</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_regMaskInputs">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>40</y>
     <width>220</width>
     <height>220</height>
    </rect>
   </property>
   <property name="title">
    <string>Inputs</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="comboBox_directDilation_ImgDim">
        <item>
         <property name="text">
          <string>2D Data</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3D Data</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4D Data</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_directDilation_imgUpl">
        <property name="text">
         <string>Open Image Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_directDilation_mskUpl">
        <property name="text">
         <string>Open Prev. Mask Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_directDilation_recUpl">
        <property name="text">
         <string>Open Reconst. Data</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_Exclusion">
        <property name="text">
         <string>Exclusion</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_directDilation_run">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>840</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_directDilation_ImgName">
      <property name="text">
       <string>File Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_directDilationImgName"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_directDilationImgSave">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
